src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/DeprecatedListWriter.java

```diff
@@ -300,6 +301,11 @@ protected void generateDeprecatedListFile(DeprecatedAPIListBuilder deprapi)
         String description = "deprecated elements";
         body.add(bodyContents);
         printHtmlDocument(null, description, body);
+
+        if (!deprapi.isEmpty() && configuration.mainIndex != null) {
+            configuration.mainIndex.add(IndexItem.of(IndexItem.Category.MEMBERS,
+                    resources.getText("doclet.Deprecated_API"), path));
+        }
```

Comment:
`IndexItem.Category.MEMBERS` is probably not the correct category to use for the Deprecated API page, and we should choose a fitting category because these summary page items will end up in the search index (see my comment in `IndexItem#add`)

Reply:
Agreed. Now that you raise the issue, I realize that we should only use MODULE/PACKAGE/TYPES/MEMBERS for the appropriate kinds of elements, and should use TAGS for all "plain text" entries.